cmd/bad-key-revoker/main.go

```diff
@@ -265,6 +284,22 @@ func (bkr *badKeyRevoker) revokeCerts(revokerEmails []string, emailToCerts map[s
 // invoke processes a single key in the blockedKeys table and returns whether
 // there were any rows to process or not.
 func (bkr *badKeyRevoker) invoke() (bool, error) {
+	// Gather a count of rows to be processed.
+	uncheckedCount, err := bkr.countUncheckedKeys()
+	if err != nil {
+		return false, err
+	}
+
+	// Set the gauge to the number of rows to be processed (max:
+	// blockedKeysGaugeLimit).
+	keysToProcess.Set(float64(uncheckedCount))
+
+	if uncheckedCount == blockedKeysGaugeLimit {
```

Comment:
```suggestion
	if uncheckedCount >= blockedKeysGaugeLimit {
```

I like to always use `>=` when it matches the semantics, even if we believe the number can never actually be `>`. Sometimes that belief turns out to be wrong, and it's nice to do the right thing anyhow.